docs/application/vscode-ext/web.md

```diff
@@ -223,88 +204,82 @@ To debug application, follow these steps:
 > **Note**
 >
 > Ensure the following:
-> - Emulator is launched
-> - Device is connected to the PC
+> - Emulator is launched, or device is connected to the PC
 > - Chrome browser is installed
 
-1. Open **Command Palette** and enter **Debug Tizen App**.
-2. Enter the name of the Tizen Web App you want to run.
-3. A successful popup is created and the app is launched.
+1. In VS Code, click **View** > **Command Palette**.
```

Comment:
For step 1 to step 7, please verify the alignment. As of now it seems to be part of note.
Please left align it, so that it aligns to the first sentence of this section:
To debug application, follow these steps:

Reply:
done.